bin/download-theme

```diff
@@ -11,9 +11,10 @@ from typing import Any
 import cson  # pip install cson
 
 TOKEN = re.compile(br'(\\\\|\\"|"|//|\n)')
+COMMAS = re.compile(br'(?<=[}\]"\']),(?!\s*[{["\'])')
```

Comment:
I think this might be too greedy and would have to be part of the `_remove_js_comments` which tokenizes things -- notably I think it has a false positive for `","`

Reply:
Already tested for the false positive

INPUT:
```python
import re
commas = re.compile(r'(?<=[}\]"\']),(?!\s*[{["\'])')
jsn = '{"boys":["antony", "bobby","fred",]}
re.sub(commas, '', jsn)
```
OUTPUT:
`'{"boys":["antony", "bobby","fred"]}'`

Care to elaborate on how you mean it being too greedy

Reply:
I mean something like this:
```json
'{"boys":["antony", ",", "bobby","fred"]}'
```